docs/installation/server-config/security.rst

```diff
@@ -159,16 +163,15 @@ non-interactive queries, see :ref:`restrict-ksql-interactive`.
 Interactive KSQL clusters
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Kafka clusters that use ACLs to control access to resources require clients to authenticate as a particular user. Interactive
-KSQL clusters require that this authenticated KSQL user has open access to create, read, write, delete topics, and use any
-consumer group.
+Interactive KSQL clusters require that the authenticated KSQL user has open access to create, read, write, delete topics, and use any
```

Comment:
"Interactive KSQL clusters require" -> "[Interactive KSQL clusters](link to section under Configuring KSQL where we explain interactive clusters) (which is the default configuration) require"